Gemfile.lock

```diff
@@ -55,6 +60,3 @@ PLATFORMS
 DEPENDENCIES
   cocoapods
   xcpretty
-
-BUNDLED WITH
```

Comment:
If you upgrade your `bundler` version this should come back. I'd prefer to still have it here since others running it will generate this part.